### PR TITLE
Improve UG according to PE feedback

### LINKED ISSUE #155: [PE-D][Tester E] Default value for status
Perhaps UG could mention the default status of interviewees is undecided for clarity.

-------------
Labels: `severity.VeryLow` `type.DocumentationBug`
original: shuenj/ped#4

### PR DESCRIPTION
**Pull request by jingting1412**
Closes #155 
Specified the default status as UNDECIDED. Specified max length of keywords for find is 55 characters

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -31,27 +32,27 @@ img
...
 * [Features](#features)
   * [Applicant Management Features](#applicant-management-features)
-    * [Command Parameters](#command-parameters-1)
-    * [Adding a new applicant: `add`](#add)
-    * [Editing an applicant: `edit`](#edit)
-    * [Deleting an applicant: `delete`](#delete)
-    * [Listing all applicants: `list`](#list)
-    * [Editing an applicant status: `status`](#status)
+    * [Command Parameters](#command-parameters)
+    * [Adding a new applicant: `add`](#adding-a-new-applicant--add)
```
Comment (ivanleekk) [inline docs/UserGuide.md:36]: hmm the link in the table of contents doesnt seem to be working in the preview

**Review by AustinHuang1203 — APPROVED** [review]
lgtm

File: docs/UserGuide.md [hunk 2]
```diff
@@ -246,18 +247,28 @@ Adds a new applicant to the list.
...
 <box type="warning" header="**Caution**">
-    Duplicate applicants are not allowed. Two applicants are considered duplicates if they have the same phone number or email.
+    Duplicate applicants are not allowed. 
+    Two applicants are considered duplicates if they have the same phone number or email.
 </box>
 
 <box type="tip" header="**Note**">
-    As <code>NAME</code> can only contain alphanumeric characters and spaces, names containing hyphens <code>-</code> or slashes <code>/</code> are not allowed in this version of the app.  
+    As <code>NAME</code> can only contain alphanumeric characters and spaces, 
+    names containing hyphens <code>-</code> or slashes <code>/</code> are not allowed in this version of the app.
```
Comment (ivanleekk) [inline docs/UserGuide.md:256]: maybe we could add that other non alpha numeric characters in general are not allowed in the app 😃

File: docs/UserGuide.md [hunk 3]
```diff
@@ -246,18 +247,28 @@ Adds a new applicant to the list.
...
+
+<box type="tip" header="**Note**">
+    Note that the default status for new Applicants is UNDECIDED.
 </box>
 
 
 Example:
 * `add n/John Doe hp/91234567 e/johndoe@gmail.com p/Software Engineer`
-* `add n/Jane Greenwood p/Project Manager e/janeg@yahoo.com hp/81234567`
+Adds a new applicant with name *John Doe*, phone number *912345671*, email *johndoe@gmail.com*, and position *Software Engineer*.
```
Comment (ivanleekk) [inline docs/UserGuide.md:266]: there is an extra 1 at the back of the phoen number here 🥸

**Review by ivanleekk — CHANGES_REQUESTED** [review]
Hi sorry just a few small changes for a typo in phone number section. Also the table of contents links dont seem to work 😢

**Review by ivanleekk — APPROVED** [review]
Nice all the stuff works now! Great work 🤩

**Pull request merged**